\documentclass[document.tex]{subfiles}
\begin{document}

\chapter{Summary of packages and methods for bioinformatics}
\label{day:2023-02-15}

\section*{Quality control + alignment metrics}

    \subsection{GenomeScaope 2.0 :\citetitle{ranallobenavidez_2020}}
    \begin{itemize}
    \item Suite of tools to measure genome quality, heterozygosity, size and repetativeness
    \item 
    \end{itemize}

    \subsection{RepeatModeller2:\citetitle{flynn_2020}}
    \begin{itemize}
    \item 
    \item 
    \end{itemize}

    \subsection{RepeatMasker :\citetitle{}}
    \begin{itemize}
    \item 
    \item 
    \end{itemize}

\section*{Genome evolution + genome architecture + varient calling}

    \subsection{PerSVade :\citetitle{schikoratamarit_2022}}
    \begin{itemize}
    \item PerSVade - package for detecting structual varients using short read data
    \item Git: https://github.com/Gabaldonlab/perSVade 
    \end{itemize}
    
    \subsection{Parliament2 :\citetitle{zarate_2020}}
    \begin{itemize}
    \item A concensus method which is used to call strutual varients
    \item docker image availble at https://github.com/slzarate/parliament2 
    \item CANNOT GET TO WORK. ERROR "cp: cannot stat '/home/dnanexus/in/None': No such file or directory" devs do not appear to be active
    \end{itemize}
    
    \subsection{LUMPY :\citetitle{layer_2014}}
    \begin{itemize}
    \item Used to call strutual varients 
    \end{itemize}

    \subsection{fastPHASE :\citetitle{}}
    \begin{itemize}
    \item Can be used to infer missing haplotypes
    \item 
    \end{itemize}

\section*{Genetic Structure and population dynamics}
    \subsection{STRUCTURE: \citetitle{pritchard_2000}}
        \begin{itemize}
        \item Utilizes an iterative Baysien algorythm to examine distribution of genetic 
        varients between "populations" and assigns samples to groups based on patterns of 
        variation.
        \item Variations:fastSTRUCTURE etc 
        \item publications of interest:
        \item Requires a minimum f 90 markers, equal sampling and exhibits greater resolution with with a higher degree of differentiation among regions \cite{nelson_2013}
        \end{itemize} 

    \subsection{AMOVA: \citetitle{excoffier_1992}}
        \begin{itemize}
        \item Analysis of Molecular Variance (AMOVA) produces estimates of variance components and F-statistic analogs, designated here as phi-statistics, reflecting the correlation of haplotypic diversity at different levels of hierarchical subdivision
        \item Can produce reliable results with fewer markers than STRUCTURE \cite{nelson_2013}
        \item Availble in GenoDive \cite{meirmans_2020}, R package/s : stats v3.6.2, poppr
        \end{itemize} 

    \subsection{BA3-SNPsv: \citetitle{mussmann_2019}}
         \begin{itemize}
        \item Modified version of BayesAss3, which is able to use SNP data
        \item Detects population migrants , dispersal rate, geentic divergence between populations
        \item Average run time of 32 hours for 10mil MCMC gen 
        \end{itemize} 

    \subsection{Evidente-a: \citetitle{wittepaz_2022}}
        \begin{itemize}
        \item Web tool which allows visualization of SNPs which support specific branches within a phylogeny
        \item Most useful for small genomes such as bacteria
        \end{itemize}
            
    \subsection{GenoDive: \citetitle{meirmans_2020}}
        \begin{itemize}
        \item GUI for analysis of population genetic data
        \item Included: AMOVA, STRUCTURE, VEGAN, INSTRUCT, LFMM etc
        \end{itemize} 

\section*{Environmental variables and Local adaptaton}

    \subsection{ECODIST: \citetitle{goslee_2007}}
        \begin{itemize}
        \item R package which uses a dissimilarity based analysis of ecological data 
        \item Mantel test
        \item ordianation and cluster analyses
        \end{itemize} 


        \subsection{PCADapt v4.3.2: \citetitle{prive2020performing}}
        \begin{itemize}
        \item R package which uses PCA clustering to dwtwct outliers
        \item tutorial: https://bcm-uga.github.io/pcadapt/articles/pcadapt.html
        \item 
        \end{itemize} 

        \subsection{BayeScan: \citetitle{foll2008genome}}
        \begin{itemize}
        \item git: https://github.com/mfoll/BayeScan
        \item Based on the multinomial-Dirichlet model
        \item Uses differences in allele frequencies between "population" so detect outlier/candidate loci
        \end{itemize} 

        \subsection{Outflank \citetitle{whitlock2015reliable}}
        \begin{itemize}
        \item Detection of Fst outlier loci using an inferred neurtral distribution
        \item site: http://rstudio-pubs-static.s3.amazonaws.com/305384_9aee1c1046394fb9bd8e449453d72847.html
        \item 
        \end{itemize} 

        \subsection{RDA \citetitle{vegan: Community Ecology Package}}
        \begin{itemize}
        \item tutorial: https://r.qcbs.ca/workshop10/book-en/redundancy-analysis.html
        \item 
        \item 
        \end{itemize} 

        \subsection{LDA \citetitle{}}
        \begin{itemize}
        \item Linear Discriminant Analysis (LDA) "Linear discriminant analysis (LDA) is a constrained (canonical) technique that divides a response matrix into groups according to a factor by finding combination of the variables that give best possible separation between groups. The grouping is done by maximizing the among-group dispersion versus the within-group dispersion. This allows you to determine how well your independent set of variables explains an a priori grouping, which may have been obtained from a previous clustering analysis (see Workshop 9) or from a hypothesis (e.g. grouping is based on sites at different latitudes or different treatments)."
        \item 
        \item 
        \end{itemize} 

        \subsection{MEM/MSOD \citetitle{}}
        \begin{itemize}
        \item Spatial detection of outlier loci with Moran eigenvector maps (MEM). Can be used to detect outliers without environmental data
        \item https://popgen.nescent.org/2016-12-13_MEM_outlier.html
        \item 
        \end{itemize} 

        \subsection{ \citetitle{}}
        \begin{itemize}
        \item 
        \item 
        \item 
        \end{itemize} 

        \subsection{ \citetitle{}}
        \begin{itemize}
        \item 
        \item 
        \item 
        \end{itemize} 


        Pop-Gen Pipeline Platform (PPP) 



\section*{Divergence time estimation and RASP}

BEAST2
RASP


\section*{Mapping}


\section*{Databases}

    \subsection{Rat Genome Database RGD: \citetitle{smith_2020}}
    \begin{itemize}
    \item https://rgd.mcw.edu/
    \end{itemize} 

    \subsection{Global Biodiversity Information Facility (GBIF): \citetitle{smith_2020}}
    \begin{itemize}
    \item R.novegicus sighting database ref: GBIF.org (20 February 2023) GBIF Occurrence Download  https://doi.org/10.15468/dl.gjbddq
    \item R.rattus sighting database ref: GBIF.org (20 February 2023) GBIF Occurrence Download https://doi.org/10.15468/dl.z8n5xq
    \end{itemize} 

    

\bib{}
    \end{document}